arch/x86/core/x86_mmu.c

```diff
@@ -8,6 +8,13 @@
 #include <mmustructs.h>
 #include <linker/linker-defs.h>
 
+/* Macro needed to do appropriate typecasting based on the paging mode used.*/
+#ifdef CONFIG_X86_PAE_MODE
+#define X86_PAGE_ENTRY_data_TYPE u64_t
```

Comment:
Stray `data_`?  Given the comment on `fatal.c`:252: was this code compiled with both PAE enabled and disabled?

Reply:
My local merge issue. was using this in the previous patch set. Removed it and created a typedef in mmustructs.h
I will also get rid of this faulty macro.

Reply:
still seems to be here